packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1134,7 +1134,7 @@ mixin TextInputClient {
   ///   * [sendAppPrivateCommand](https://developer.android.com/reference/android/view/inputmethod/InputMethodManager#sendAppPrivateCommand),
   ///     which is the Android documentation for sendAppPrivateCommand, used to
   ///     send a command to the input method.
-  void performPrivateCommand(String action, Map<String, dynamic> data);
+  void performPrivateCommand(String action, Map<String, dynamic>? data);
```

Comment:
This breaks internal tests I think?

Reply:
I'll try running the Google tests on this PR when it goes green.  I thought about just creating an empty map if it's null and keeping this parameter the same non-nullable, but it felt dirty.  I'll consider just migrating any breakages myself if it's not too crazy.